docs/faq.md

```diff
@@ -0,0 +1,252 @@
+
+### <a name="default-mode"></a> What is the default mode when performing optimization?
+
+The default mode is min when performing optimization. The mode can be configured when instantiating a scheduler.
+
+### <a name="trial-execution"></a> How are trials evaluated when evaluating trials on a local machine?
+
+When trials are executed locally (e.g. when `LocalBackend` is used), each trial is evaluated as a different sub-process.
+
+### <a name="tuning-output"></a> What does the the output of the tuning contain?
+
+When running locally, Syne Tune store tuning results on  `~/syne-tune/{tuner-name}/`. This directory will contain `metadata.json`, `results.csv.zip`, and `tuner.dill` which are respectively metadata of the tuning job, results obtained at each time-step and state of the tuner.
+
+When running remotely, those files are saved by default under `s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/`.
+
+### <a name="trial-checkpointing"></a> How can I enable trial checkpointing?
+
+Since trials may be paused and resumed (either by schedulers or when using spot-instances), the user has the possibility to checkpoint intermediate results to avoid starting computation from scratch. Model outputs and checkpoints must be written into a specific local path given by the command line argument `st_checkpoint_dir`. Saving/loading model checkpoint from this directory enables to save/load the state when the job is stopped/resumed (setting the folder correctly and uniquely per trial is the responsibility of the backend), see [checkpoint_example.py](https://github.com/awslabs/syne-tune/blob/main/examples/training_scripts/checkpoint_example/checkpoint_example.py) to see a fully working example of a tuning script with checkpoint enabled.
+
+When using SageMaker backend or tuning remotely, we use [SageMaker checkpoint mechanism](https://docs.aws.amazon.com/sagemaker/latest/dg/model-checkpoints.html) under the hood to sync local checkpoints to s3. Checkpoints are synced to  `s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/{trial-id}/`, where `sagemaker-default-bucket` is the default bucket for SageMaker (can also be configured).
+
+### <a name="schedulers-checkpointing"></a> Which schedulers make use of checkpointing?
+
+Checkpointing means storing the state of a trial (i.e., model parameters, optimizer or learning rate scheduler parameters), so that it can be paused and potentially resumed at a later point in time, without having to start training from scratch. Syne Tune checkpointing support is detailed in https://github.com/awslabs/syne-tune/blob/main/docs/benchmarks.md#checkpointing and https://github.com/awslabs/syne-tune/blob/main/examples/training_scripts/checkpoint_example/checkpoint_example.py. Once checkpointing to and from a local file is added to a training script, Syne Tune is managing the checkpoints (e.g., copy to/from cloud storage for distributed back-ends).
```

Comment:
Nicer links: [here](https://github.com/awslabs/syne-tune/blob/main/docs/benchmarks.md#checkpointing) and [checkpoint_example.py](https://github.com/awslabs/syne-tune/blob/main/examples/training_scripts/checkpoint_example/checkpoint_example.py)

Reply:
I am a bit surprised these are all absolute links. Don't we want to use relative links, not starting with https://github.com ?